### PR TITLE
Upload to local pypi

### PR DESCRIPTION
**Pull request by eyal-stratoscale**
@Stratoscale/mgmt

### CODE REVIEW

File: hammock/cli/command.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def get_parser(self, prog_name):
     def _action(self, parsed_args):
         kwargs = {arg: getattr(parsed_args, arg) for arg in self.spec.all_args}
         if self.spec.keywords:
-            kwargs.update(getattr(parsed_args, self.spec.keywords, ''))
+            kwargs.update(getattr(parsed_args, self.spec.keywords, None) or {})
```
Comment (shoham-stratoscale) [inline hammock/cli/command.py]: `kwargs.update(getattr(parsed_args, self.spec.keywords, {}))`

File: Makefile [hunk 2]
```diff
@@ -44,12 +44,17 @@ rename: dist/hammock-rest-0.0.1-1.noarch.rpm
 	mv $< $(basename $<)-$(shell git rev-parse --short=7 HEAD).rpm
 
 upload:
```
Comment (shoham-stratoscale) [inline Makefile]: why submit doesn't call uploads?

File: hammock/cli/command.py (hunk 1 above)
Comment (eyal-stratoscale) [inline hammock/cli/command.py]: I want to change it to empty dict also if it is None

File: Makefile (hunk 2 above)
Comment (eyal-stratoscale) [inline Makefile]: do you think it should be there?
lets talk about it

**Comment by shoham-stratoscale** [comment]
shipit

File: Makefile [hunk 3]
```diff
@@ -32,24 +32,22 @@ unittest:
...
 
-rename: dist/hammock-rest-0.0.1-1.noarch.rpm
-	- rm -f $(basename $<)-*.rpm
-	mv $< $(basename $<)-$(shell git rev-parse --short=7 HEAD).rpm
-
 upload:
-	python setup.py sdist upload
+	python setup.py sdist upload -r http://rackattack-nas.dc1:5001
 
 submit:
```
Comment (ravid-stratoscale) [inline Makefile]: Don't we want the submit to call upload?

**Pull request merged**